packages/clarity-decode/src/interaction.ts

```diff
@@ -72,6 +61,14 @@ export function decode(tokens: Data.Token[]): InteractionEvent {
                 y: tokens[4] as number,
             };
             return { time, event, data: scrollData };
+        case Data.Event.Timeline:
+            let timelineData: Interaction.TimelineData = {
+                type: tokens[2] as number,
+                target: tokens[3] as number,
```

Comment:
How would we represent things like click text, clicked link in this model?

Reply:
It will still be part of the normal event. This only captures the summary.